Build/Customizations/UpdateBuildScriptValue.targets

```diff
@@ -19,7 +19,6 @@
   <PropertyGroup>
     <RemotionBuildFile Condition="'$(RemotionBuildFile)' == ''">$(SolutionDirectory)\BuildScript\Main\Remotion.build</RemotionBuildFile>
     <AssemblyNuGetVersion></AssemblyNuGetVersion>
-    <Version></Version>
```

Comment:
change why?

Reply:
To me it looks like this: *.targets files are imported after *.props files, so this overrides the used version to Empty, making the build fail. I'm guessing this is a bug that nobody noticed?

Reply:
I'm unsure of what setting Version to empty was supposed to achieve in the first place

Reply:
> 
> 
> I'm unsure of what setting Version to empty was supposed to achieve in the first place

Just declaring the "variable" / "parameter". It wasn't meant to set to blank.

Reply:
> 
> 
> To me it looks like this: *.targets files are imported after *.props files (Version is set to 2.0.0 in Version.props), so this overrides the used version to Empty, making the build fail. I'm guessing this is a bug that nobody noticed?

Yes, props before code, then targets at the end is the import order normally implemented by MSBuild scripts.

Just noticed, we're talking about something from the build script customizations in the build for this repository. yeah, okay, probably something weird/incorrect going on. Just make it good :)